---
output:
  pdf_document: 
      latex_engine: xelatex
      keep_tex: true
  html_document: default
  word_document: default
bibliography: "`r here::here('bib/THESIS.bib')`"
always_allow_html: true
---

# Identifying protrudin's neuronal functions

```{=latex}
\chaptermark{Neuronal functions}
```

```{r chapter_2_setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)

knitr::knit_child(here::here("analysis/Protrudin_spine_type_analysis.Rmd"))
knitr::knit_child(here::here("analysis/Protrudin_spine_length_analysis.Rmd"))
knitr::knit_child(here::here("analysis/Protrudin_proteomics.Rmd"))
knitr::knit_child(here::here("analysis/HeLa_screen.Rmd"))
knitr::knit_child(here::here("analysis/Protrudin_autophagy_analysis.Rmd"))

library(knitr)
library(kableExtra)

options(knitr.table.format = 'markdown')
```

## Introduction

As discussed previously in section \@ref(intro-ptdn), protrudin's array of binding domains allow it to participate in numerous cellular systems, with diverse effects.  To identify other mechanisms of action, we decided to take a broad approach, examining a number of different systems to identify how protrudin might influence neuronal function, either endogenously or with overexpression.

Previous work in the lab found it extremely difficult to reduce protrudin levels in neurons — either by shRNA or CRISPR — without killing the cells.  We were able to study loss of protrudin expression through overexpression of mutants with dominant negative effects, rather than protrudin deletion or silencing.  Since our plasmids drive protrudin overexpression at levels substantially higher than endogenous protrudin expression [@petrovaProtrudinFunctionsEndoplasmic2020], overexpression of a mutant form of protrudin should outcompete the endogenous protein for binding partners, even if it does not directly decrease the amount of endogenous protein in the cell.  In this way, we are able to examine the effects of different loss-of-function mutations on protrudin's cellular effects.

### Cellular morphology

While protrudin's role in cell outgrowth is interesting from the perspective of axon regeneration, it could potentially play a number of other roles in neuron morphology.  Petrova et al. showed that endogenous protrudin is present in both axons and dendrites, but is most abundant in the dendrites and cell bodies of mature neurons [@petrovaProtrudinFunctionsEndoplasmic2020].  Consequently, protrudin's endogenous functions might be more apparent in the somatodendritic domain, simply due to the natural distribution of the protein.  In fact, there are a number of published findings that suggest that protrudin may function in dendrites, either to control dendritic spine morphology or support synaptic function.  

For example, the mechanism through which protrudin promotes invadopodia formation in cancer cells is functionally similar to how lysosomal exocytosis induces dendritic spine expansion, raising the possibility that protrudin is involved in spine growth [@pedersenProtrudinmediatedEREndosome2020; @padamseyActivityDependentExocytosisLysosomes2017].  However, Pedersen et al. found that this process is dependent on FYCO1 as well as protrudin, which (as discussed in section \@ref(no-FYCO)), is largely absent in CNS neurons [@pedersenProtrudinmediatedEREndosome2020].  On the other hand, protrudin has already been shown to play a functional role at dendritic spines, mediating long term depression by allowing internalised AMPA receptors to be removed from the spine through association with KIF5 [@brachetKinesin1protrudinComplex2021].  In addition, ER tubules (or the more complicated spine apparatus) have been observed either transiently or permanently in most spines on hippocampal neurons [@perez-alvarezEndoplasmicReticulumVisits2020], which also suggests that protrudin is likely to be present in or near spines and could act on their morphology.  Given protrudin's effect in cellular outgrowth, therefore, it is reasonable to expect that protrudin overexpression might have an effect on dendritic spine morphology.

### Functional effects

To further examine protrudin's endogenous roles beyond cellular outgrowth, we turned to the literature to identify published proteomics data sets.  We identified three studies that published lists of protrudin-interacting proteins [@elbaz-alonPDZD8InteractsProtrudin2020; @hashimotoProtrudinRegulatesEndoplasmic2014; @matsuzakiProtrudinServesAdaptor2011].  Each study used a different model system and focused their analysis on different pathways, providing a variety of perspectives.  

Elbaz-Alon et al. looked at protrudin in a human epithelial cell line, where they investigated how protrudin interacts with PDZD8 and contributes to ER contact sites  [@elbaz-alonPDZD8InteractsProtrudin2020].  Hashimoto et al. used mouse brain neurons to look at interactions between protrudin and HSP-related proteins [@hashimotoProtrudinRegulatesEndoplasmic2014], while Matsuzaki et al. used a mouse neuroblast cell line to study the interaction between protrudin and KIF5 [@matsuzakiProtrudinServesAdaptor2011].  Individually, these reveal a number of interesting interactions, indicating diverse roles for protrudin that include but are not limited to axonal function.  Taken together, a substantial level of overlap between the interaction sets reveals a conserved set of cellular functions that may help explain protrudin's role in neurons.

### Experimental aims

Following the results discussed in the previous chapter, we decided to temporarily shift focus away from axonal transport and investigate how protrudin might affect other cellular properties.  First, we examined dendritic spines to see if protrudin overexpression increased the number or size of spines, and whether protrudin loss-of-function mutants showed an opposite effect.  We then analysed published protrudin proteomics data to identify previously unstudied organelles and pathways that might be affected by protrudin overexpression, and followed this up with a screen in HeLa cells to select markers for neuronal experiments.



## Methods

General cell culture and microscopy methods can be found in section \@ref(METHODS).

### Dendritic spine imaging and analysis

Neurons were co-transfected with mCherry or an mCherry-tagged protrudin construct and EGFP/Emerald.  Cells were imaged as Z-stacks at DIV15, either on the spinning disc at 100x, or confocal at 63x (see \@ref(METHODS) for microscope details).  Image stacks from the green channel were converted to maximum intensity projections, and a custom ImageJ macro was used to select and crop 20 \textmu{}m sections from a dendrite.  These images were then blinded, and the Cell Counter Fiji plugin was used to count and classify spines.  For spine length, the Fiji segmented line tool was used to measure along each spine.

### Gene ontology analysis

Lists of protrudin-interacting proteins were obtained from supplementary data published with Elbaz-Alon et al., Hashimoto et al., and Matsuzaki et al. [@elbaz-alonPDZD8InteractsProtrudin2020;@hashimotoProtrudinRegulatesEndoplasmic2014;@matsuzakiProtrudinServesAdaptor2011].  Gene names were standardised across experiments (and between human and mouse data sets) using Bioconductor genome annotations (`org.Hs.eg.db` version 3.15.0 and `org.Mm.eg.db` version 3.15.0).  Gene ontology analysis for biological process enrichment was carried out using the `limma` Bioconductor package in R.

### HeLa cell culture

HeLa cells were expanded before use and cryopreserved in individual vials to prevent genetic drift between batches.

HeLa cells were grown in DMEM *(Thermo Fisher, 41966-029)*, supplemented with 10% FBS *(Thermo Fisher, 10500064)* and antibiotic-antimycotic *(Thermo Fisher, 15240062)*, at 37°C, 5% CO~2~.  For each batch of cells, a single vial was thawed, passaged, and grown to 60-80% confluence before transfection.  Cells were transfected with EGFP or EGFP-tagged protrudin under a CAG promoter, using TransIT-LT1 reagent *(Mirus Bio, MIR2300)*, according to the manufacturer's protocol.  These were then passaged 24 hours after transfection to reduce overlap between cells and encourage protrusion growth.

Cells were fixed with 3% formaldehyde in PBS for 15 minutes at room temperature.  Immunochemistry was carried out according to the protocol in section \@ref(immuno).  Staining with BODIPY TR C~5~ ceramide *(Thermo Fisher, B34400)* was carried out according to the manufacturer's protocol.  Briefly, cells were incubated with 5 \textmu{}M dye in DMEM for 30 minutes at 4°C.  They were then recovered in supplemented media at 37°C, 5% CO~2~ for 30 minutes before fixing as usual.

## Results

### Protrudin does not influence dendritic spine number, length, or morphology

Given protrudin's dendritic localisation and known effects on protrusion formation, we first decided to examine whether protrudin plays a role in regulating dendritic spine morphology.  Dendritic spines show diverse morphology within and between cells, which is generally considered to be reflective of the maturity, functionality, and plasticity of each spine.  While these ultimately fall on a continuous spectrum of shapes, they are conventionally grouped into four morphological classes — mushroom, thin, stubby, and filopodia [@pchitskayaDendriticSpinesShape2020].  We decided to use these classes to obtain an overview of spine morphology on cells expressing protrudin.  

We co-transfected rat cortical neurons with EGFP/Emerald (as a cytoplasmic label) and an mCherry-labelled protrudin construct.  These included WT and active protrudin, as well as a set of domain-deletion mutants to act as dominant negatives.  \textDelta{}FFAT, \textDelta{}RBD, and \textDelta{}FYVE each have a single binding motif deleted or disrupted, \textDelta{}KIF5 has both the CC and FFAT domains deleted, to completely block binding to kinesin, and \textDelta{}ER mutant (\textDelta{}TM1-3) has all transmembrane domains deleted [@petrovaProtrudinFunctionsEndoplasmic2020]. (Figure \@ref(fig:mutant-maps-full-figure))

```{r mutant-maps-full-figure, echo=F, fig.width=16, fig.height = 16, dpi = 160, fig.path = '../figures/full/', fig.cap='Domain maps of the protrudin constructs used in this experiment.  Not to scale.', fig.scap="Domain maps of protrudin mutants."}
image_as_ggplot(read.bitmap(here('figures/external/protrudin_mutant_maps.png')))
```

For each of these conditions, we counted and manually classified dendritic spines on 20 \textmu{}m sections of dendrite from cells at DIV15 or DIV21.  The DIV15 data suggests that WT protrudin might be associated with fewer, stubbier spines, although the significant comparisons varied for each spine classification, and is inconsistent with the DIV21 data. (Figure \@ref(fig:spine-type-full-figure))

```{=latex}
\newgeometry{a4paper, top=40mm, left=35mm, right=35mm, bottom = 30mm} %unnecessary as long as 1.5in margins overall
```
```{r spine-type-full-figure, echo = F,  fig.width = 16, fig.height = 22, fig.path = '../figures/full/', fig.cap="A) Schematic showing selection of dendrite sections for analysis and basic spine morphology groups.  One dendrite section was analysed per cell.  Scale bar is 10 \\textmu{}m.  B-C) Total spine counts in 20 \\textmu{}m section for DIV15 and DIV21 groups.  D-E) Quantification of each spine type, normalised to the total number of spines in the section measured.  DIV15: n = 30 cells per condition across three independent batches; DIV21: n = 10 cells per condition in one batch.", fig.scap="Protrudin effect on dendritic spine morphology"}

spine_types_plot
```
```{=latex}
\restoregeometry
\setstretch{1.5} %not restored by \restoregeometry
```

As manual spine classification is an inherently subjective measure, and did not show any clear associations, we decided to look instead at spine length, which can be measured more objectively.  We re-analysed the initial DIV15 data, measuring the length of each spine from the edge of the dendrite.  This did yield some significant comparisons, both in spine number and length, so we followed this up with a full replication, which we analysed in the same way (figure \@ref(fig:spine-length-full-figure)).  While the replication also produced statistically significant results, they were wholly inconsistent between the two experiments.    

```{r spine-length-full-figure, echo = F, fig.width = 16, fig.height = 14, fig.path = '../figures/full/', fig.cap = "A-B) Number of dendritic spines counted per 20 \\textmu{}m section in cells co-expressing EGFP/Emerald with an mCherry-tagged protrudin construct.  Replicate 1: n = 30 cells per condition across three independent batches, except \\textDelta{}RBD n = 29, \\textDelta{}KIF5 n = 20, \\textDelta{}FYVE n = 19 across two batches.  Replicate 2: n = 40 cells per condition, except WT and \\textDelta{}ER n = 39, across three independent batches. C-D) Length of each spine measured, in \\textmu{}m.", fig.scap="Protrudin effect on dendritic spine length"}

spine_lengths_full_plot
```

Taken together, these results show that protrudin overexpression does not substantially impact dendritic spine number, length, or morphology.  These measurements were highly variable between cells and between replicates, and did not show any consistent patterns that would suggest that protrudin plays any role in controlling spine outgrowth.  This obviously cannot rule out changes in spine function, but in any case such changes do not seem to be mediated by morphological changes at the micron scale.

### Gene ontology analysis of protrudin-interacting proteins{#GO-analysis}

As our results did not indicate a clear role for protrudin in regulating dendritic spine morphology, we turned to proteomic analysis to identify other potential cellular functions, either in the somatodendritic domain or in axons.

We identified three published lists of protrudin-interacting proteins from Elbaz-Alon et al., Hashimoto et al., and Matsuzaki et al. [@elbaz-alonPDZD8InteractsProtrudin2020;@hashimotoProtrudinRegulatesEndoplasmic2014;@matsuzakiProtrudinServesAdaptor2011].  As these studies used distinct methods and model systems, we were most interested in the interactions that were conserved across sets.  After standardising the data across sources, we found 46 protrudin-interacting proteins that were identified in at least two studies.  Gene ontology analysis (biological process enrichment) of this set showed a high representation of proteins involved in vesicular transport, and in particular Golgi-related transport ('intra-Golgi vesicle mediated transport', 'Golgi vesicle transport', 'ER to Golgi vesicle-mediated transport', etc.) (figure \@ref(fig:proteomics-full-figure)).   In contrast to endosomal transport, protrudin's role in Golgi-related transport has not been extensively studied, and we decided to follow this up with further experiments.

```{r proteomics-full-figure, fig.width = 16, fig.height = 6.5, fig.path = "../figures/full/", fig.cap="A) Venn diagram showing overlap between published lists of protrudin-interacting proteins. B) Top 20 hits of GO biological process enrichment, of all proteins appearing in at least two protrudin interaction sets, sorted by p-value.", fig.scap="Data set comparison and gene ontology of conserved protrudin-interacting proteins"}
proteomics_full
```

### HeLa screen for changes in organelle and protein distribution

When protrudin is overexpressed in HeLa cells, it drives the growth of long cellular protrusions that superficially resemble developing neurites [@shiraneProtrudinInducesNeurite2006].  While these cells are genetically and functionally very distinct from neurons, this nevertheless provides a useful model system for examining protrudin's interactions with other cellular components.  We used this system to screen for proteins that colocalise with protrudin in these cellular outgrowths, to quickly identify organelles or complexes that might be transported into axons when protrudin is overexpressed in neurons.

Based on the results of our gene ontology analysis, we selected a panel of antibodies to screen for colocalisation with protrudin in HeLa cells.  These were primarily focused on components of the ER exit site to Golgi secretory pathway, but also included proteins involved in lipid metabolism, signalling, and amino acid transport.

```{r}
ab = read.csv(here('data/tables/antibodies.csv'))
kable(ab[ab$HeLa_screen == TRUE & ab$stage %in% c('primary', 'dye'), c('Marker', 'Description')],
      booktabs = TRUE,
      caption = 'Markers for HeLa screen.')
```

#### ER- and Golgi-related organelles

Of the ERES-related proteins, different markers showed different levels of colocalisation with protrudin.  BCAP31 and SURF4 were consistently enriched in cellular protrusions, more so than at other areas of the cell periphery.  MIA3 and SEC31A colocalised well with protrudin in the perinuclear area, but were found at lower levels in protrusions, and were less consistent between cells.  At this stage, it was not possible to conclusively say whether these variations represented differences in how ERES proteins were transported or in how effectively the individual antibodies stained these components, so we decided to proceed with BCAP31 and SURF4, which produced more clear results. (Figure \@ref(fig:HeLa-ERES-grid-full-figure))

```{r HeLa-ERES-grid-full-figure, fig.width=16, fig.height=16, fig.path = "../figures/full/", fig.cap="Representative images of HeLa cells expressing EGFP-tagged protrudin constructs and stained for ER/ERES-related markers A) BCAP31, B) SURF4, C) MIA3(TANGO1), and D) SEC31A.  All scale bars are 20 \\textmu{}m.", fig.scap = "Protrudin-expressing HeLa cells stained for BCAP31, SURF4, MIA3, and SEC31A"}
ERES_grid
```

Of the Golgi-related proteins, GS28, GLG1, and USO1 clearly labelled the perinuclear Golgi compartment, but were either entirely absent or found only at very low levels in protrusions.  ERGIC53^[ERGIC53, or LMAN1, is the human homolog of a protein sometimes known as p58 in rat and several other species [@lahtinenMolecularCloningExpression1996].  For the sake of simplicity I will refer to it as ERGIC53 throughout this thesis.], on the other hand, was highly enriched in protrusions, in most cases well above the levels seen in the cell body.  This was the strongest concentration with any of the markers used in this screen, and point to the ERGIC compartment not only colocalising with protrudin, but specifically playing a role in the growth of cellular protrusions.  This marker was added to the set (along with BCAP31 and SURF4) to examine in neurons.  (Figure \@ref(fig:HeLa-Golgi-grid-full-figure))

```{r HeLa-Golgi-grid-full-figure, fig.width=16, fig.height=22, fig.path = "../figures/full/", fig.cap="Representative images of HeLa cells expressing EGFP-tagged protrudin constructs and stained for ERGIC and Golgi-related markers A) ERGIC53, B) GS28, C) GLG1, and D) USO1(p115)  E) Close up of ERGIC53 staining in the protrusion of a cell expressing WT protrudin, as shown in panel A.  F) Fluorescence profile along the protrusion shown in panel E.  All scale bars are 20 \\textmu{}m.", fig.scap = "Protrudin-expressing HeLa cells stained for ERGIC53, GS28, GLG1, and USO1"}
Golgi_extended
```

#### Lysosomes, enzymes, and transporters

LAMP1 and mTOR, which we expected to move to the cell periphery in response to protrudin overexpression, were present in protrusions at relatively low levels.  While protrudin overexpression did result in more puncta away from the perinuclear cluster, this effect was not especially dramatic, and the highest concentrations were still found near the cell nucleus.  This is in contrast with [@hongPtdIns3PControlsMTORC12017], which found a stronger movement away from the nucleus when overexpressing Myc-tagged protrudin.  However, their culture protocol differed from ours in adding 2 mM supplemental glutamine to the culture media, so the apparent differences could reflect the amino acid dependence of the protrudin-FYCO1 transport mechanism.  (Figure \@ref(fig:HeLa-misc-grid-full-figure), panels A-B)

ACSL3 was found in cellular protrusions, but — similar to MIA3 and SEC31A — was generally at low levels, and varied considerably between cells.  This may reflect on the antibody used, or may represent more complicated cell-state dependent interaction with other metabolic pathways.  This was not taken forwards.  (Figure \@ref(fig:HeLa-misc-grid-full-figure), panel C)

Finally, SLC38A1 was found to concentrate in protrusions, and similarly to ERGIC53 was found at higher levels here than near the nucleus.  This result is particularly interesting, as SLC38A1 is a potential cargo of the local secretory pathway.  Matsuzaki et al. identified SLC38A1 in their set of protrudin-interacting proteins, and Shigeoka et al. showed that it was locally translated in RGC axons [@matsuzakiProtrudinServesAdaptor2011; @shigeokaDynamicAxonalTranslation2016]. SLC38A1 was added to the list with BCAP31, SURF4, and ERGIC53 to examine in neurons.  (Figure \@ref(fig:HeLa-misc-grid-full-figure), panel D)

```{r HeLa-misc-grid-full-figure, fig.width=16, fig.height=16, fig.path = "../figures/full/", fig.cap="Representative images of HeLa cells expressing EGFP-tagged protrudin constructs and stained  for A) LAMP1, B) mTOR, C) ACSL3, and D) SLC38A1.  All scale bars are 20 \\textmu{}m.", fig.scap = "Protrudin-expressing HeLa cells stained for LAMP1, mTOR, ACSL3, and SLC38A1"}
misc_grid
```

#### Other morphological effects of protrudin overexpression

In addition to protrusions, we noted that protrudin overexpression also induces the formation of large, ring-shaped structures in the perinuclear region of HeLa cells.  These are observed occasionally in neurons expressing protrudin, but not to the same extent as in HeLa cells.  While these were not the primary focus of this experiment, they did reveal interesting results with two of the markers screened, and may indicate protrudin involvement in additional cellular pathways.

LC3 strongly colocalised with protrudin in these structures, suggesting that they may represent phagophores or autophagosomes (figure \@ref(fig:LC3-full-figure)).  Notably, LC3 is *not* enriched in cellular protrusions (panel C), in contrast to the markers discussed above.  This may represent a different pathway activated by protrudin overexpression.  These LC3-labelled structures are also smaller and less abundant in cells expressing only EGFP, suggesting that their formation is influenced by protrudin overexpression.

We followed up this result with a pilot experiment in neurons, co-expressing EGFP-protrudin and mRFP1-tagged LC3.  LC3 showed differences in localisation between cells, ranging from fully cytoplasmic to fully punctate, which likely reflects the conversion between soluble LC3-I and membrane-associated LC3-II [@kabeyaLC3MammalianHomologue2000].  All cells expressing either WT or active protrudin contained at least some LC3 puncta, in contrast to control cells, of which 12.5% had fully cytoplasmic LC3.  This suggests that protrudin might be encouraging autophagosome biogenesis, although further experiments are needed to confirm this result.

```{r LC3-full-figure, fig.width=16, fig.height=18, fig.path = "../figures/full/", fig.cap="A) Representative image of HeLa cells expressing WT protrudin and stained for LC3.  Inset is 20x20 \\textmu{}m.  B) Fluorescence profile along the dotted line shown in panel A inset.  Note strong correspondence between protrudin (green) and LC3 (magenta) signal.  C) Protrusion of a HeLa cell expressing WT protrudin and stained for LC3.  Note the lack of LC3 enrichment with protrudin in the tip, compared to the cell body.  D) Representative image of a HeLa cell expressing GFP and stained for LC3.  Note the lack of large, ring-shaped structures as seen in the protrudin expressing cell in panel A.  E) Representative images of neurons expressing mRFP1-tagged LC3 showing range of distribution patterns.  F) Quantification of LC3 distributions in cells co-expressing EGFP-protrudin and mRFP1-LC3.  n = 40 cells per condition across two independent batches.  All scale bars are 20 \\textmu{}m.", fig.scap="Protrudin effect on LC3 localisation in HeLa cells and cortical neurons"}
LC3_full_plot
```

BODIPY C~5~ ceramide, a lipid dye often used to label the Golgi apparatus, strongly labelled the lumen of these organelles.   It is not clear in this context whether the dye is labelling Golgi-related structures or lipid droplets, however it is worth noting that other Golgi markers did *not* colocalise with these structures (e.g. figure \@ref(fig:HeLa-Golgi-grid-full-figure), particularly visible in panel C).  Given the relationship with LC3 as discussed above, this may indicate an increase in lipophagy, although further experiments would be needed to confirm this hypothesis.

```{r ceramide-full-figure, fig.width = 16, fig.height = 12, fig.path = "../figures/full/", fig.cap="A) Representative image of HeLa cells expressing WT protrudin and stained with BODIPY C$\\textsubscript{5}$ ceramide dye.  Inset is 20x20 \\textmu{}m.  B) Fluorescence profile along the dotted line shown in panel A inset.  Note BODIPY signal (magenta) between spikes of protrudin (green).  C) Representative image of a HeLa cell expressing GFP and stained with BODIPY C$\\textsubscript{5}$ ceramide dye.  Note the lack of large, bright puncta as seen in the protrudin-expressing cell in panel A.  All scale bars are 20 \\textmu{}m.", fig.scap = "Protrudin effect on BODIPY C$\\textsubscript{5}$ ceramide localisation in HeLa cells"}
ceramide_grid
```

## Discussion

### Protrudin's effect on dendritic spine morphology

Protrudin overexpression — or inhibition through the expression of dominant negative mutants — did not substantially influence dendritic spine morphology. Considering the mechanism described in Pedersen et al. (in which protrudin-mediated cellular outgrowth is dependent on interaction with Rab7 and FYCO1), it seems likely that this is another result of the absence of FYCO1 in CNS neurons [@pedersenProtrudinmediatedEREndosome2020].  We cannot, however, rule out an effect on dendritic spine function, and more work would be required to understand whether protrudin overexpression may influence neuronal signalling.

### The ERES to Golgi pathway

In non-polarised cells, membrane proteins are typically translated in the ER, released from ERES, trafficked to the Golgi via the ERGIC compartment, and secreted in post-Golgi carriers.  In neurons, several alternate pathways exist, to account for the local translation needs of compartments distant from the cell body.  In dendrites, for example, Golgi outposts and satellites serve some of the same functions as the main Golgi apparatus, acting as local points for protein glycosylation [@mikhaylovaGS2016].  The ERGIC compartment is also involved in a Golgi-bypassing secretory pathway, which has been observed in dendrites [@bowenGolgiindependentSecretoryTrafficking2017].  

There is evidence for the local translation of integral membrane proteins in CNS axons [@shigeokaDynamicAxonalTranslation2016], although it is not currently understood how they are trafficked from the ER to the cell surface.  Recent work in DRG neurons has identified a non-conventional secretory pathway involving Golgi satellites and LAMP1-labelled organelles, which plays a role in the delivery of TRPM8 ion channels [@cornejoNonconventionalAxonalOrganelles2020].  It is possible that a similar pathway functions in CNS axons, providing a mechanism through which protrudin expression could promote secretion and growth in the absence of any changes to the cell body Golgi apparatus.  

It is worth noting that three of the four proteins identified in our HeLa screen — ERGIC53, SURF4, and BCAP31 — have been identified in the ERGIC compartment, although there is some indication that BCAP31 localisation varies by cell type [@breuzaProteomicsEndoplasmicReticulumGolgi2004; @quistgaardBAP31PhysiologicalFunctions2021].  This result suggests a novel role for this compartment in protrudin-driven protrusion formation, and potentially in axon regeneration, which is explored further in the next chapter.
