---
output:
  word_document: default
  html_document: default
  pdf_document: 
      latex_engine: xelatex
      keep_tex: true
bibliography: "`r here::here('bib/THESIS.bib')`"
---

# Protrudin's role in regeneration and endogenous functions

```{r chapter_2_setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)

knitr::knit_child(here::here("analysis/Protrudin_spine_type_analysis.Rmd"))
knitr::knit_child(here::here("analysis/Protrudin_spine_length_analysis.Rmd"))
knitr::knit_child(here::here("analysis/Protrudin_proteomics.Rmd"))
knitr::knit_child(here::here("analysis/HeLa_screen.Rmd"))
knitr::knit_child(here::here("analysis/Protrudin_autophagy_analysis.Rmd"))

library(knitr)

```

## Introduction
As discussed previously (CROSS_REF TO INTRO), protrudin's array of binding domains allow it to participate in numerous cellular systems, with diverse effects.  To identify other mechanisms of action, we decided to take a broad approach, examining a number of different systems to identify how protrudin might influence survival and regeneration in neurons.


### Cellular morphology
While protrudin's role in cell outgrowth is interesting from the perspective of axon regeneration, it could potentially play a number of other roles in neuron morphology  For example, the mechanism through which protrudin promotes invadopodia formation [@pedersenProtrudinmediatedEREndosome2020] is functionally very similar to how lysosomal exocytosis induces dendritic spine expansion, raising the possibility that protrudin is involved in spine morphology.  However, [@pedersenProtrudinmediatedEREndosome2020] found that this process is dependent on FYCO1 as well as protrudin, which (as established in (REF BACK TO CH1)), is largely absent in CNS neurons.
 
On the other hand, protrudin has already been shown to play a functional role at dendritic spines, mediating long term depression by allowing internalised AMPA receptors to be removed from the spine through association with KIF5 [@brachetKinesin1protrudinComplex2021].  In addition, ER tubules (or the more complicated spine apparatus) have been observed either transiently or permanently in most spines on hippocampal neurons [@perez-alvarezEndoplasmicReticulumVisits2020], which suggests that protrudin is likely to be present in or near spines and could act on their morphology.  Given protrudin's effect in cellular outgrowth, therefore, it is reasonable to expect that protrudin overexpression might have an effect on dendritic spine morphology.

### Functional effects
To examine protrudin's role beyond cellular outgrowth, we turned to the literature to identify published proteomics data sets.  We identified three studies that published full lists of protrudin-interacting proteins [@elbaz-alonPDZD8InteractsProtrudin2020][@hashimotoProtrudinRegulatesEndoplasmic2014][@matsuzakiProtrudinServesAdaptor2011].  While each of these studies used a different model system and focused their analysis on different pathways, there was substantial overlap between the sets, representing a conserved set of cellular functions that may help explain how protrudin promotes survival and regeneration.




## Methods

General cell culture and microscopy methods can be found in section \@ref(METHODS).

### Dendritic spine imaging and analysis
Neurons were co-transfected with mCherry or an mCherry-tagged protrudin construct and EGFP/Emerald.  Cells were imaged as Z-stacks at DIV15, either on the spinning disc at 100x, or confocal at 63x (see \@ref(METHODS) for microscope details).  Image stacks from the green channel were converted to maximum intensity projections, and a custom ImageJ macro was used to select and crop 20$\mu$m sections from a dendrite.  These images were then blinded, and the Cell Counter Fiji plugin was used to count and classify spines.  For spine length, the Fiji segmented line tool was used to measure along each spine.

### Gene ontology analysis
Lists of protrudin-interacting proteins were obtained from supplementary data published with [@elbaz-alonPDZD8InteractsProtrudin2020], [@hashimotoProtrudinRegulatesEndoplasmic2014], and [@matsuzakiProtrudinServesAdaptor2011].  Gene names were standardised across experiments (and between human and mouse data sets) using Bioconductor genome annotations.  Gene ontology analysis for biological process enrichment was carried out using the *limma* Bioconductor package in R.

### HeLa cell culture
HeLa cells were expanded before use and cryopreserved in individual vials to prevent genetic drift between batches.

HeLa cells were grown in DMEM *(Thermo Fisher, 41966-029)*, supplemented with 1% FBS *(Thermo Fisher, 10500064)* and antibiotic-antimycotic *(Thermo Fisher, 15240062)*.  For each batch of cells, a single vial was thawed, passaged, and grown to 60-80% confluence before transfection.  Cells were transfected with EGFP or EGFP-tagged protrudin under a CAG promoter, using TransIT-LT1 reagent *(Mirus Bio, MIR2300)*, according to the manufacturer's protocol.  These were then passaged 24h after transfection to reduce overlap between cells and encourage protrusion growth.

Cells were fixed with 3% formaldehyde in PBS for 15 mintues at room temperature.  Immunochemistry was carried out according to the protocol in (REF TO SPECIFIC METHODS SECTION).



## Results
### Protrudin's effect on dendritic spine number, length, and morphology
Dendritic spines show diverse morphology within and between cells, which is generally considered to be reflective of the maturity, functionality, and plasticity of each spine.  While these ultimately fall on a continuous spectrum of shapes, they are conventionally grouped into four morphological classes - mushroom, thin, stubby, and filopodia [@pchitskayaDendriticSpinesShape2020].  I decided to use these classes to obtain an overview of spine morphology on cells expressing protrudin.  

I counted and manually classified dendritic spines on 20$\mu$m sections of dendrite from DIV15 (n = 30 cells per condition across three batches) or DIV21 neurons (n = 10 cells per condition, one batch) co-expressing an EGFP/Emerald filler with an mCherry-labelled protrudin construct (\@ref(fig:spine_type_full_figure)).  These included WT and active protrudin, as well as a set of domain-deletion mutants.  $\Delta$FFAT, $\Delta$RBD, and $\Delta$FYVE each have a single binding motif deleted, $\Delta$KIF5 has both the CC and FFAT domains deleted, to completely block binding to kinesin, and $\Delta$ER mutant has all transmembrane domains deleted [@petrovaProtrudinFunctionsEndoplasmic2020].  The DIV15 data suggests that WT protrudin might be associated with fewer, stubbier spines, although the significant comparisons varied for each spine classification, and is inconsistent with the DIV21 data.

```{r spine_type_full_figure, echo = F, fig.width = 16, fig.height = 22, fig.path = '../figures/full/', fig.cap="A) Schematic showing selection of dendrite sections for analysis and basic spine morphology groups.  One dendrite section was analysed per cell.  Scale bar is 10$\\mu$m.  B-C) Total spine counts in 20$\\mu$m section for DIV15 and DIV21 groups.  D-E) Quantification of each spine type, normalised to the total number of spines in the section measured.  DIV15: n=30 cells per condition across three independent batches; DIV21: n=10 cells per condition in one batch."}

spine_types_plot
```

As manual spine classification is an inherently subjective measure, and did not show any clear associations, I decided to look instead at spine length, which can be measured more objectively.  I re-analysed the initial DIV15 data, measuring the length of each spine from the edge of the dendrite.  This did yield some significant comparisons, both in spine number and length, so I followed this up with a full replication, which I analysed in the same way.  While the replication also produced statistically significant results, they were wholly inconsistent between the two experiments.    

```{r spine_length_full_figure, echo = F, fig.width = 16, fig.height = 14, fig.cap = "A-B) Number of dendritic spines counted per 20$\\mu$m section in cells co-expressing EGFP/Emerald with an mCherry-tagged protrudin construct.  Replicate 1: n=30 cells per condition across three independent batches, except $\\Delta$RBD n = 29, $\\Delta$KIF5 n = 20, $\\Delta$FYVE n = 19 across two batches.  Replicate 2: n = 40 cells per condition, except WT and $\\Delta$ER n = 39, across three independent batches. C-D) Length of each spine measured, in $\\mu$m. TODO - GET N VALUES FOR SPINES"}

spine_lengths_full_plot
```


Taken together, these results show that protrudin overexpression does not substantially impact dendritic spine number, length, or morphology.  These measurements were highly variable between cells and between replicates, and did not show any consistent patterns that would suggest that protrudin plays any role in controlling spine outgrowth.  This obviously cannot rule out changes in spine function, but in any case such changes do not seem to be mediated by morphological changes at the micron scale.




### Protrudin's interactions with other proteins in the cell

We identified three published lists of protrudin-interacting proteins from [@elbaz-alonPDZD8InteractsProtrudin2020], [@hashimotoProtrudinRegulatesEndoplasmic2014], and [@matsuzakiProtrudinServesAdaptor2011].  As these studies used distinct methods and model systems, we were most interested in the interactions that were conserved across sets.  After standardising the data across sources, we found 46 protrudin-interacting proteins that were identified in at least two studies.  Gene ontology analysis (biological process enrichment) of this set showed a high representation of genes involved in vesicular transport, and in particular Golgi-related transport ('intra-Golgi vesicle mediated transport', 'Golgi vesicle transport', 'ER to Golgi vesicle-mediated transport', etc.) (REF TO FIGURE).   In contrast to ensosomal transport, protrudin's role in Golgi-related transport has not been extensively studied, and we decided to follow this up with further experiments.

```{r proteomics_full_figure, fig.width = 16, fig.height = 6, fig.path = "../figures/full/", fig.cap="A) Venn diagram showing overlap between published lists of protrudin-interacting proteins. B) Top 20 hits of GO biological process enrichment, of all proteins appearing in at least two protrudin interaction sets, sorted by p-value."}
proteomics_full
```



### HeLa screen for changes in organelle distribution

When protrudin is overexpressed in HeLa cells, it drives the growth of long cellular protrusions that superficially resemble developing neurites [@shiraneProtrudinInducesNeurite2006].  While these cells are genetically and functionally very distinct from neurons, this nevertheless provides a useful model system for examining protrudin's interactions with other cellular components.  I used this system to screen for proteins that colocalise with protrusion in these cellular outgrowths, to quickly identify organelles or complexes that could be transported into axons when protrudin is overexpressed in neurons.

(INSERT MARKER/ANTIBODY TABLE HERE)


Based on the results of our gene ontology analysis, we selected a panel of antibodies to screen for colocalisation with protrudin in HeLa cells.  These were primarily focused on components of the ER exit site to Golgi secretory pathway, but also included proteins involved in lipid metabolism, signalling, and amino acid transport.

```{r HeLa_ERES_grid_full_figure, fig.width=16, fig.height=16, fig.path = "../figures/full/", fig.cap="Representative images of HeLa cells expressing EGFP-tagged protrudin constructs and stained for ER/ERES-related markers A)BCAP31, B)SURF4, C)MIA3(TANGO1), and D)SEC31A.  All scale bars are 20$\\mu$m."}
ERES_grid
```

Of the ERES-related proteins, different markers showed different levels of colocalisation with protrudin.  BCAP31 and SURF4 were consistently enriched in cellular protrusions, more so than at other areas of the cell periphery.  MIA3 and SEC31A colocalised well with protrudin in the perinuclear area, but were found at lower levels in protrusions, and were less consistent between cells.  At this stage, it was not possible to conclusively say whether these variations represented differences in how ERES proteins were transported or in how effectively the individual antibodies stained these components, so we decided to proceed with BCAP31 and SURF4, which produced more clear results.

```{r HeLa_Golgi_grid_full_figure, fig.width=16, fig.height=16, fig.path = "../figures/full/", fig.cap="Representative images of HeLa cells expressing EGFP-tagged protrudin constructs and stained for ERGIC and Golgi-related markers A)ERGIC53, B)GS28, C)GLG1, and D)USO1(p115)  All scale bars are 20$\\mu$m."}
Golgi_grid
```

Of the Golgi-related proteins, GS28, GLG1, and USO1 clearly labelled the perinuclear Golgi compartment, but were either entirely absent or found only at very low levels in protrusions.  ERGIC53, on the other hand, was highly enriched in protrusions, in most cases well above the levels seen in the cell body.  This was the strongest concentration with any of the markers used in this screen, and point to the ERGIC compartment not only colocalising with protrudin, but specifically playing a role in the growth of cellular protrusions.  This marker was added to the set (along with BCAP31 and SURF4) to examine in neurons.

```{r HeLa_misc_grid_full_figure, fig.width=16, fig.height=16, fig.path = "../figures/full/", fig.cap="Representative images of HeLa cells expressing EGFP-tagged protrudin constructs and stained  for A)LAMP1, B)mTOR, C)ACSL3, and D)SLC38A1.  All scale bars are 20$\\mu$m."}
misc_grid
```

LAMP1 and mTOR, which we would expect to move to the cell periphery in response to protrudin overexpression, were present in protrusions at relatively low levels.  While protrudin overexpression did result in more puncta away from the perinuclear cluster, this effect was not especially dramatic, and the highest concentrations were still found near the cell nucleus.  This is in contrast with [@hongPtdIns3PControlsMTORC12017], which found a stronger movement away from the nucleus when expressing Myc-tagged protrudin.  However, their culture protocol differed from ours in adding 2mM supplemental glutamine to the culture media, so the apparent differences could reflect the amino acid dependence of the protrudin-FYOC1 transport mechanism.

ACSL3, was found in cellular protrusions, but, similar to MIA3 and SEC31A, was generally at low levels, and varied considerably between cells.  This may reflect on the antibody used, or a more complicated cell-state dependent interaction with other metabolic pathways.  This was not taken forwards.

Finally, SLC38A1, was found to concentrate in protrusions, and similarly to ERGIC53 was found at higher levels here than near the nucleus.  This result is particularly interesting, as SLC38A1 is a potential cargo of the local secretory pathway.  SLC38A1 was added to BCAP31, SURF4, and ERGIC53 to examine in neurons.



In addition to protrusions, protrudin overexpression also induces the formation of large, ring-shaped structures in the perinucear region.  These are observed occasionally in neurons expressing protrudin, but not to the same extent as in HeLa cells.  While these were not the primary focus of this experiment, they did reveal interesting results with two of the markers screened, and may indicate protrudin involvement in non-morphological pathways.

LC3 strongly colocalised with protrudin in these structures, suggesting that they may represent phagophores or autophagosomes (LINK TO FIGURE).  Notably, LC3 is *not* enriched in cellular protrusions (LINK TO FIGURE PANEL), in contrast to the markers discussed above.  This may represent a different pathway activated by protrudin overexpression.  These LC3-labelled structures are also smaller and less abundant in cells expressing only EGFP, suggesting that their formation is influenced by protrudin overexpression.

I followed up this result with a pilot experiment in neurons, co-expressing EGFP-protrudin and mRFP1-tagged LC3.  LC3 showed differences in localisation between cells, ranging from fully cytoplasmic to fully punctate, which likely reflects the conversion between soluble LC3-I and membrane-associated LC3-II (CITE).  All cells expressing either WT or active protrudin contained at least some LC3 puncta, in contrast to control cells, of which 12.5% had fully cytoplasmic LC3.  This suggests that protrudin might be encouraging autophagosome biogenesis, although further experiments are needed to confirm this result.

```{r LC3_full_figure, fig.width=16, fig.height=12, fig.path = "../figures/full/", fig.cap="A) Representative image of HeLa cells expressing WT protrudin and stained for LC3.  Inset is 20x20$\\mu$m.  B)Flurescence profile along the dotted line shown in panel A inset.  Note strong correspondence between protrudin (green) and LC3 (magenta) signal.  C)Representative images of neurons expressing mRFP1-tagged LC3 showing range of distribution patterns.  Scale bars are 20$\\mu$m.  D)Quantification of LC3 distributions in cells co-expressing EGFP-protrudin and mRFP1-LC3.  n = 40 cells per condition across two independent batches."}
LC3_full_plot
```

BODIPY C~5~ ceramide, a lipid dye often used to label the Golgi apparatus, strongly labelled the lumen of these organelles.   It is not clear in this context whether these are Golgi-related structures or lipid droplets, however it is worth noting that other Golgi markers did *not* colocalise with these structures (e.g. REF TO GOLGI FIGURE, PANEL C GLG1).  Given the relationship with LC3 as discussed above, this may indicate an increase in lipophagy, although further experiments would be needed to confirm.

```{r ceramide_full_figure, fig.width = 16, fig.height = 8, fig.path = "../figures/full/", fig.cap="A)Representative image of HeLa cells expressing WT protrudin and stained with BODIPY C~5~ ceramide dye.  Inset is 20x20$\\mu$m.  B)Fluorescence profile along the dotted line shown in panel A inset.  Note BODIPY signal (magenta) between spikes of protrudin (green)."}
ceramide_grid
```

## Discussion

### The ERES to Golgi pathway

In non-polarised cells, proteins are typically translated in the ER, released from ERES, trafficked to the Golgi via the ERGIC compartment, and secreted in post-Golgi carriers.  In neurons, several alternate pathways exist, to account for the local translation needs of compartments distant from the cell body.  In dendrites, for example, Golgi outposts and satellites serve some of the same functions as the main Golgi apparatus, acting as local points for protein glycosylation [@mikhaylovaDendriticGolgiSatellite2016].  The ERGIC compartment is also involved in a Golgi-bypassing secretory pathway, which has been observed in dendrites [@bowenGolgiindependentSecretoryTrafficking2017].  It is possible that that same mechanisms could function in axons, providing a mechanism through which protrudin expression could promote secretion and growth in the absence of any changes to the cell body Golgi apparatus.

